server/routes/api.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const router = express.Router()
+const request = require('request')
+
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756",
+    "wizards": "1610612764",
+    "raptors": "1610612761",
+    "spurs": "1610612759",
+    "rockets": "1610612745"
+}
+
+const dreamTeam = [];
+
+let allPlayers;
+
+router.get(`/teams/:teamName`, function (req, res) {
+    const { teamName } = req.params;
+    const teamId = teamToIDs[teamName]
+    request('http://data.nba.net/10s/prod/v1/2018/players.json', function (error, result) {
+        players = JSON.parse(result.body).league.standard
+        players = players.filter(player => player.teamId == teamId && player.isActive).map(player => {
+            return { firstName: player.firstName, lastName: player.lastName, jersey: player.jersey, position: player.pos }
+        })
+        players.forEach(a => {
```

Comment:
You are writing code that is not needed. You were able to write it in previous loops (no need for more loops/maps when you can write it in one loop (-saves memory).